misc/cleanup_dupes.sql

```diff
@@ -0,0 +1,29 @@
+CREATE OR REPLACE PROCEDURE cleanup_dupes()
+LANGUAGE 'plpgsql'
+AS $$
+DECLARE
+    c_before bigint;
+	c_after bigint;
+BEGIN
+	RAISE NOTICE 'Rows before cleanup %.', (SELECT COUNT(*) FROM atoms_b);
+    c_before = (SELECT COUNT(DISTINCT data_json) FROM atoms_b);
+	
+	-- Delete dupes
+	DELETE FROM
+    atoms_b a USING atoms_b b
+	WHERE a.id < b.id
```

Comment:
Why not something more complex? What if one of the atoms is contained in another one?
[This trello board](https://trello.com/c/57EQA2xD) contains a few queries.

```
WHERE a.id < b.id
   AND b.data_json <@ a.data_json;
```

By checking "is contained" you could be removing distinct values (if the two atoms are not equal) and the distinct count of values check wouldn't work.

IDK I'm fine with this cleanup method too, just consider the is contained check and let me know.